test/integration/connect/envoy/case-grpc/verify.bats

```diff
@@ -26,10 +26,6 @@ load helpers
   [ "$status" == 0 ]
 }
 
-@test "s1 proxy should be sending gRPC metrics to statsd" {
-  run retry_default must_match_in_statsd_logs 'envoy.cluster.default.dc1.internal.*.consul.grpc.PingServer.total'
-
-  echo "OUTPUT: $output"
-
-  [ "$status" == 0 ]
-}
+@test "gateway-primary is used for the upstream connection" {
```

Comment:
This is `case-grpc` so should this comment mention gateways?

Reply:
something got mixed up here, this should not have changed so I must have changed the wrong file somewhere.